infrastructure/azure.tf

```diff
@@ -16,177 +16,196 @@ provider "null" {
 variable "azurerm_cosmosdb" {
-    type = "string"
+  type = "string"
 }
 
 variable "azurerm_cosmosdb_documentdb" {
-  type = "string"
+  type        = "string"
   description = "Name of CosmosDB Database"
 }
 
 variable "azurerm_cosmosdb_collections" {
-  type = "map"
+  type        = "map"
   description = "Name and partition keys of collections that must exist in the CosmosDB database"
 }
 
 # Name of the App Service Plan resource
 variable "azurerm_app_service_plan" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the App Service Plan for developer portal
 variable "azurerm_app_service_plan_portal" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the App Service for developer portal
 variable "azurerm_app_service_portal" {
-    type = "string"
+  type = "string"
 }
 
 variable "app_service_portal_git_repo" {
-  type = "string"
+  type        = "string"
   description = "URL of the GitHub repository providing the source of the App Service Portal"
 }
 
 variable "app_service_portal_git_branch" {
-  default = "master"
+  default     = "master"
   description = "Branch of the GitHub repository providing the source of the App Service Portal"
 }
 
 # Redirect to this page after developer portal login
 variable "app_service_portal_post_login_url" {
-    type = "string"
+  type = "string"
 }
 
 # Redirect to this page after developer portal logout
 variable "app_service_portal_post_logout_url" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the API management resource
 variable "azurerm_apim" {
-    type = "string"
+  type = "string"
+}
+
+variable "azurerm_apim_sku" {
+  type = "string"
 }
 
 # Name of the ADB2C policy
 variable "azurerm_adb2c_policy" {
-    type = "string"
+  type = "string"
+}
+
+# TF_VAR_ADB2C_TENANT_ID
+variable "ADB2C_TENANT_ID" {
+  type = "string"
+}
+
+# TF_VAR_DEV_PORTAL_CLIENT_ID
+variable "DEV_PORTAL_CLIENT_ID" {
+  type = "string"
+}
+
+# TF_VAR_DEV_PORTAL_CLIENT_SECRET
+variable "DEV_PORTAL_CLIENT_SECRET" {
+  type = "string"
 }
 
 # Name of Application Insights resource
 variable "azurerm_application_insights" {
-    type = "string"
+  type = "string"
 }
 
 # Name of Log Analytics resource
 variable "azurerm_log_analytics" {
-    type = "string"
+  type = "string"
 }
 
 # EventHub namespace
 variable "azurerm_eventhub_ns" {
-    type = "string"
+  type = "string"
 }
 
 # EventHub logger for API management
 variable "azurerm_apim_eventhub" {
-    type = "string"
+  type = "string"
 }
 
 # EventHub rule for API management
 variable "azurerm_apim_eventhub_rule" {
-    type = "string"
+  type = "string"
 }
 
 variable "azurerm_shared_address_space_cidr" {
-  default = "100.64.0.0/10"
+  default     = "100.64.0.0/10"
   description = "Azure internal network CIDR"
 }
 
 variable "azurerm_azure_portal_ips" {
-  default = "104.42.195.92,40.76.54.131,52.176.6.30,52.169.50.45,52.187.184.26"
+  default     = "104.42.195.92,40.76.54.131,52.176.6.30,52.169.50.45,52.187.184.26"
```

Comment:
actually something I wanted to ask, is there a link to a page that lists these IPs that we can add here as reference?

Reply:
https://docs.microsoft.com/en-us/azure/cosmos-db/firewall-support